src/client/i18n.js

```diff
@@ -16,8 +16,7 @@ i18n
   // for all options read: https://www.i18next.com/overview/configuration-options
   .init({
     fallbackLng: "en",
-    debug: false,
-
+    debug: true,
```

Comment:
Is it ok to commit this change? Will this change behavior in production?

Reply:
good point, i'm not certain so I'll set it to true on dev only